lib/syncthing/utils.go

```diff
@@ -124,5 +125,9 @@ func copyFile(src, dst string) error {
 }
 
 func OpenGoleveldb(path string, tuning config.Tuning) (*db.Lowlevel, error) {
-	return db.Open(path, db.Tuning(tuning))
+	ldb, err := backend.Open(path, backend.Tuning(tuning))
+	if err != nil {
+		return nil, err
+	}
+	return db.NewLowlevel(ldb), nil
```

Comment:
I'd like to use this change to make the interface of `lib/syncthing` deal with `backend.Backend` instead of `*db.Lowlevel`. Or do you want to defer that to a separate PR.

Reply:
I'm not sure we want to expose backend outside the db package. Can we explore later?